### ISSUE TITLE
[E4] Duda Mastermind

### ISSUE THREAD

**Issue by martina-abello** [franciscogazitua/IIC2113-2024#208]
Cuando dice "el total de bonus de la unidad", eso considera los bonus de primer ataque y follow up? gracias de antemano.

**Comment by cahinostroza** [comment]
No, sólo se consideran los bonus que no son aplicados a una fase del combate específica.
